src/polymer-cli.ts

```diff
@@ -42,6 +43,21 @@ export class PolymerCli {
   }
 
   run(args) {
+    // If the "--version"/"-v" flag is ever present, just print
+    // the current version. Useful for globally installed CLIs.
+    if (args.indexOf('--version') > -1 || args.indexOf('-v') > -1) {
+      console.log(packageJSON.version);
```

Comment:
Yes, but in this case it's really `require` abuse to load a file. It's not a library import, it's a data file, which can't have dependencies, is not listed in package.json, etc. It doesn't need to be loaded at startup at all.